routes/books.js

```diff
@@ -28,7 +28,7 @@ router.route("/").post(function (req, res) {
 router
   .route("/:id")
   .get(function (req, res) {
-    db.books.get(req.params.id).then((book) => {
+    db.books.get(req.params.id, req.session.userId).then((book) => {
```

Comment:
You've changed the signature for the `books.get` db function to take 3 parameters, and this only gives 2.